Assets/Scripts/UIAndMenus/EscMenuController.cs

```diff
@@ -13,26 +18,37 @@ void Start()
     {
         pauseMenuPanel.SetActive(false);
         settingsMenuPanel.SetActive(false);
+        DontDestroyOnLoad(gameObject);
```

Comment:
![image](https://github.com/user-attachments/assets/a9cbdbc7-9c8e-4827-a217-8a39de60be43)

Unity is throwing a warning saying that this line will not work--you should probably check this is doing what you want it to